Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/gauffrea/.vscode/extensions/ms-python.debugpy-2024.10.0-linux-x64/bundled/libs/debugpy/adapter/../../debugpy/launcher/../../debugpy/__main__.py", line 39, in <module>
    cli.main()
  File "/home/gauffrea/.vscode/extensions/ms-python.debugpy-2024.10.0-linux-x64/bundled/libs/debugpy/adapter/../../debugpy/launcher/../../debugpy/../debugpy/server/cli.py", line 430, in main
    run()
  File "/home/gauffrea/.vscode/extensions/ms-python.debugpy-2024.10.0-linux-x64/bundled/libs/debugpy/adapter/../../debugpy/launcher/../../debugpy/../debugpy/server/cli.py", line 284, in run_file
    runpy.run_path(target, run_name="__main__")
  File "/home/gauffrea/.vscode/extensions/ms-python.debugpy-2024.10.0-linux-x64/bundled/libs/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_runpy.py", line 321, in run_path
    return _run_module_code(code, init_globals, run_name,
  File "/home/gauffrea/.vscode/extensions/ms-python.debugpy-2024.10.0-linux-x64/bundled/libs/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_runpy.py", line 135, in _run_module_code
    _run_code(code, mod_globals, init_globals,
  File "/home/gauffrea/.vscode/extensions/ms-python.debugpy-2024.10.0-linux-x64/bundled/libs/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_runpy.py", line 124, in _run_code
    exec(code, run_globals)
  File "/home/gauffrea/Projects/SMABalancingGroups/pretrain.py", line 94, in <module>
    if __name__ == "__main__":
  File "/home/gauffrea/Projects/SMABalancingGroups/pretrain.py", line 62, in main
    loaders = get_loaders(config.data_path, config.dataset, config.batch_size, config.method, config.SMA, transform=transform)
  File "/home/gauffrea/Projects/SMABalancingGroups/datasets.py", line 406, in get_loaders
    dataset_tr = Dataset(data_path, "tr", subsample_what, duplicates, args_SMA,custom_transform=transform)
  File "/home/gauffrea/Projects/SMABalancingGroups/datasets.py", line 143, in __init__
    super().__init__(split, root, metadata_csv_path, transform, subsample_what, duplicates)
  File "/home/gauffrea/Projects/SMABalancingGroups/datasets.py", line 28, in __init__
    self.count_groups()
  File "/home/gauffrea/Projects/SMABalancingGroups/datasets.py", line 40, in count_groups
    self.nb_labels = len(set(self.y))
NameError: name 'seltransformf' is not defined
Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/gauffrea/.vscode/extensions/ms-python.debugpy-2024.10.0-linux-x64/bundled/libs/debugpy/adapter/../../debugpy/launcher/../../debugpy/__main__.py", line 39, in <module>
    cli.main()
  File "/home/gauffrea/.vscode/extensions/ms-python.debugpy-2024.10.0-linux-x64/bundled/libs/debugpy/adapter/../../debugpy/launcher/../../debugpy/../debugpy/server/cli.py", line 430, in main
    run()
  File "/home/gauffrea/.vscode/extensions/ms-python.debugpy-2024.10.0-linux-x64/bundled/libs/debugpy/adapter/../../debugpy/launcher/../../debugpy/../debugpy/server/cli.py", line 284, in run_file
    runpy.run_path(target, run_name="__main__")
  File "/home/gauffrea/.vscode/extensions/ms-python.debugpy-2024.10.0-linux-x64/bundled/libs/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_runpy.py", line 321, in run_path
    return _run_module_code(code, init_globals, run_name,
  File "/home/gauffrea/.vscode/extensions/ms-python.debugpy-2024.10.0-linux-x64/bundled/libs/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_runpy.py", line 135, in _run_module_code
    _run_code(code, mod_globals, init_globals,
  File "/home/gauffrea/.vscode/extensions/ms-python.debugpy-2024.10.0-linux-x64/bundled/libs/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_runpy.py", line 124, in _run_code
    exec(code, run_globals)
  File "/home/gauffrea/Projects/SMABalancingGroups/pretrain.py", line 94, in <module>
    if __name__ == "__main__":
  File "/home/gauffrea/Projects/SMABalancingGroups/pretrain.py", line 62, in main
    loaders = get_loaders(config.data_path, config.dataset, config.batch_size, config.method, config.SMA, transform=transform)
  File "/home/gauffrea/Projects/SMABalancingGroups/datasets.py", line 406, in get_loaders
    dataset_tr = Dataset(data_path, "tr", subsample_what, duplicates, args_SMA,custom_transform=transform)
  File "/home/gauffrea/Projects/SMABalancingGroups/datasets.py", line 143, in __init__
    super().__init__(split, root, metadata_csv_path, transform, subsample_what, duplicates)
  File "/home/gauffrea/Projects/SMABalancingGroups/datasets.py", line 28, in __init__
    self.count_groups()
  File "/home/gauffrea/Projects/SMABalancingGroups/datasets.py", line 40, in count_groups
    self.nb_labels = len(set(self.y))
NameError: name 'seltransformf' is not defined